packages/duration/duration.0.1.0/descr

```diff
@@ -0,0 +1,6 @@
+Conversions to various time units
+
+
```

Comment:
that's the fault of topkg (or mine to not customize or adjust my `README.md` habits). anyways, fixed in next commit